### PR TITLE
Team 4 sprint 3 - Added Unlockable attires functionality + Gold_6 attire Buff/Debuff animation

### PR DESCRIPTION
**Pull request by prateek45**

### CODE REVIEW

File: source/core/assets/images/mpc/finalAtlas/gold_6_buff_to_be_tested/mpcAnimation_6.atlas [hunk 1]
```diff
@@ -0,0 +1,1196 @@
...
+  offset: 0, 0
+  index: 2
+main_player_attack_dizzy
+  rotate: false
+  xy: 3805, 7605
+  size: 541, 541
+  orig: 541, 541
+  offset: 0, 0
+  index: 1
+main_player_attack_dizzy_2.png
```
Comment (kprohith) [inline source/core/assets/images/mpc/finalAtlas/gold_6_buff_to_be_tested/mpcAnimation_6.atlas:35]: Error over here? Take a look at the sprite naming convention for main_player_attack_dizzy

Comment (kprohith) [reply source/core/assets/images/mpc/finalAtlas/gold_6_buff_to_be_tested/mpcAnimation_6.atlas:35]: @prateek45 @RecklessPotcover

Comment (prateek45) [reply source/core/assets/images/mpc/finalAtlas/gold_6_buff_to_be_tested/mpcAnimation_6.atlas]: @kprohith The bug is fixed now.

**Review by kprohith — APPROVED** [review]
Approved.

**Pull request merged**